src/main/resources/messages.properties

```diff
@@ -52,5 +53,9 @@ exception.invalid.dto=Invalid data transfer object
 exception.sql=Exception during saving object to the database
 exception.entity.not-found=Entity not found into the database
 exception.illegal.argument=Exception during request. Illegal or inappropriate argument
+exception.illegal.operation=Operation is not allowed
+exception.illegal.password.change=You are unable to change your password
+exception.illegal.username.change=You are unable to change your username
```

Comment:
You should make it universal:
`exception.illegal.field.change=You are unable to change your ${validatedValue}`